hash/ibm5170.xml

```diff
@@ -18278,6 +18540,50 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="laurabw2de" cloneof="laurabw2">
+		<description>Roberta Williams' Laura Bow in "The Dagger of Amon Ra (Germany)"</description>
```

Comment:
Shouldn't the country name fall outside the quotation marks here?

Reply:
Of course it should. Correction made.
thx!